icu4c/source/python/icutools/databuilder/test/filtration_test.py

```diff
@@ -60,12 +65,33 @@
     "zh"
 ]
 
+
+class TestIO(object):
+    def __init__(self):
+        pass
+
+    def read_locale_deps(self, tree):
```

Comment:
Is this a golden file test running with real data? If so, please raise a ticket to not make it do that.